tutorials/erb_template.md

```diff
@@ -0,0 +1,129 @@
+\page erbtemplate ERB Template
```

Comment:
Nitpick: Keep all lines with less than 80~100 chars. That makes it easier to review and to see diffs.